src/cache/cache.jl

```diff
@@ -266,3 +243,36 @@ function build_cache(
 
     return AtmosCache{map(typeof, args)...}(args...)
 end
+
+function compute_coriolis(ᶜcoord, ᶠcoord, params)
+    if eltype(ᶜcoord) <: Geometry.LatLongZPoint
+        Ω = CAP.Omega(params)
+        global_geom = Spaces.global_geometry(axes(ᶜcoord))
+        if global_geom isa Geometry.DeepSphericalGlobalGeometry
+            @info "using deep atmosphere"
+            coriolis_deep(coord::Geometry.LatLongZPoint) = Geometry.LocalVector(
+                Geometry.Cartesian123Vector(zero(Ω), zero(Ω), 2 * Ω),
+                global_geom,
+                coord,
+            )
+            ᶜf³ = @. CT3(CT123(coriolis_deep(ᶜcoord)))
+            ᶠf¹² = @. CT12(CT123(coriolis_deep(ᶠcoord)))
+        else
+            wvec = map(ᶜcoord) do coord
+                Ω = CAP.Omega(params)::eltype(coord)
```

Comment:
Can `Ω` be moved outside of the map?

Reply:
No, that will turn this into a captured variable over `Ω`.

Reply:
Right! Now that you say it makes perfect sense, but my first instinct would have been to move it outside because it is a constant.

Reply:
Yeah, for inference the scope is the important piece, and ideally we do not reach for anything outside the anonymous function (`do coord ... end`) to determine the return type inside the anonymous function.